pkg/lib/indexer/indexer.go

```diff
@@ -363,6 +363,9 @@ func (i ImageIndexer) getDatabaseFile(workingDir, fromIndex, caFile string, skip
 
 	dbLocation, ok := labels[containertools.DbLocationLabel]
 	if !ok {
+		if _, ok := labels[containertools.ConfigsLocationLabel]; ok {
```

Comment:
I think it might be worth adding a regression test

Reply:
Working on it! +1